### PR TITLE
Update Hono to 1.7.0

### PR DESCRIPTION
**Pull request by calohmn**
- Replace deprecated `ADMIN_HTTP_PORT` Jaeger all-in-one property and update Jaeger images to 1.22
- Update Infinispan to 11.09
- Add tenant service configuration for command router component (needed in Hono 1.7.0) and update Hono to 1.7.0

### CODE REVIEW

**Comment by sophokles73** [comment]
FMPOV we should postpone this to once 1.7.0 has been released. It then should also contain the switch in the appVersion.

**Comment by calohmn** [comment]
This PR contains the complete Hono 1.7 update now.

File: charts/hono/templates/hono-service-command-router/hono-service-command-router-deployment.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        {{- include "hono.component.springEnv" ( merge $args ( dict "additionalProfile" ( ( or .Values.dataGridSpec .Values.dataGridExample.enabled ) | ternary "" "embedded-cache" ) ) ) | indent 8 }}
+        {{- include "hono.component.springEnv" $args | indent 8 }}
```
Comment (sophokles73) [inline charts/hono/templates/hono-service-command-router/hono-service-command-router-deployment.yaml:52]: can we also please enable Quarkus based images for the Command Router? This simply requires adding
`"useImageType" true` to the *args* dictionary in the first line of this file ...

File: charts/hono/templates/hono-service-command-router/hono-service-command-router-secret.yaml [hunk 2]
```diff
@@ -60,16 +60,16 @@ stringData:
...
-        hostnameVerificationRequired: false
+      registration:
+        {{- include "hono.deviceRegistryExampleClientConfig" $args | nindent 8 }}
+      tenant:
+        {{- include "hono.deviceRegistryExampleClientConfig" $args | nindent 8 }}
       {{- else }}
+      registration:
         {{- required ".Values.adapters.deviceRegistrationSpec MUST be set if example Device Registry is disabled" .Values.adapters.deviceRegistrationSpec | toYaml | nindent 8 }}
+      tenant:
+        {{- required ".Values.adapters.tenantSpec MUST be set if example Device Registry is disabled" .Values.adapters.tenantSpec | toYaml | nindent 8 }}
```
Comment (sophokles73) [inline charts/hono/templates/hono-service-command-router/hono-service-command-router-secret.yaml:72]: can we also please add configuration for Quarkus logging as we do for e.g. the AMQP adapter?

**Review by sophokles73 — APPROVED** [review]
LGTM

**Comment by calohmn** [comment]
@sophokles73 I think when already doing a minor version jump, we can change the `useCommandRouter` default to be `true` here. I've just made that change and adapted the README accordingly. Can you have another look?

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**